### DISCUSSION TITLE
how use “drawBoundingBoxes” and “getWrappedImage” to show the image?

### DISCUSSION THREAD

**Discussion by xuhongxin** [deepjavalibrary/djl#2869] [Development]
![image](https://github.com/deepjavalibrary/djl/assets/1044109/992f6230-19f1-45ad-ad2e-ee5b016977ca)

i have successfully load the model,but when i want display the result , i can not see the photo result like the demo;in my opinion,use “drawBoundingBoxes” and “getWrappedImage” will display the visiable result,but it does't work;

![image](https://github.com/deepjavalibrary/djl/assets/1044109/ee929e10-c3da-493e-80fb-458f0d82e727)

**Comment by frankfliu** [comment]
Which part doesn't work? Does the jupyter notebook work for you: https://github.com/deepjavalibrary/djl-demo/blob/master/jupyter/paddlepaddle/paddle_ocr_java_zh.ipynb?

**Reply by xuhongxin** [reply]
I have built the code and run it in IntelliJ Idea;And “drawBoundingBoxes” and “getWrappedImage” seems can not
 display the photo show in the demo；but i use Image.save can get the output result photo in my local; i just want to display the image ☺

**Comment by frankfliu** [comment]
`Image.getWrappedImage()` returns `BufferedImage` object, and the jupyter notebook will render it and show it in browser.

If you want to get image in Java, you have to save the image to a file:

```
Path imagePath = Paths.get("/download/myimage.png");
image.save(Files.newOutputStream(imagePath), "png");
```

**Reply by xuhongxin** [reply]
thank you
